### ISSUE TITLE
Windows CI/CD cannot build

### ISSUE THREAD

**Issue by almightydreamer** [flutter/flutter#149619]
### Steps to reproduce

```yaml
name: Flutter Desktop CI/CD

on:
  push:
    branches:
      - testingcicd
  pull_request:
    branches:
      - testingcicd

jobs:
  build:
    name: Build
    runs-on: windows-latest

    outputs:
      app_version: ${{ steps.extract_version.outputs.app_version }}

    steps:
      - uses: actions/checkout@v3

      - name: Ensure PowerShell and where are in PATH
        run: |
          echo "Ensuring required paths are in PATH"
          $env:PATH += ";C:\Windows\System32;C:\Windows\System32\WindowsPowerShell\v1.0"
          [System.Environment]::SetEnvironmentVariable('PATH', $env:PATH, [System.EnvironmentVariableTarget]::Process)
          where.exe powershell
          where.exe cmd

      - name: Install Chocolatey
        run: |
          Set-ExecutionPolicy Bypass -Scope Process -Force; `
          [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; `
          iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))

      - name: Install Flutter SDK
        uses: subosito/flutter-action@v2
        with:
          flutter-version: '3.19.6'
          channel: "stable"

      - name: Echo Process Environment Variables
        shell: pwsh
        run: |
          Write-Output "Process PATH:"
          $path = $env:PATH
          $chunks = 50
          for ($i = 0; $i -lt $path.Length; $i += $chunks) {
          Write-Output $path.Substring($i, [Math]::Min($chunks, $path.Length - $i))
          }

      - name: Echo System Environment Variables
        shell: pwsh
        run: |
          Write-Output "System PATH:"
          $systemPath = [System.Environment]::GetEnvironmentVariable('PATH', [System.EnvironmentVariableTarget]::Machine)
          $chunks = 50
          for ($i = 0; $i -lt $systemPath.Length; $i += $chunks) {
            Write-Output $systemPath.Substring($i, [Math]::Min($chunks, $systemPath.Length - $i))
          }


      - name: Install Visual Studio Build Tools with Windows 10 SDK
        run: |
          choco install visualstudio2019buildtools --package-parameters "--add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --includeRecommended --includeOptional"

      - name: Verify Windows 10 SDK Installation
        run: |
          dir "C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0"

      - name: Install dependencies and generate models
        run: |
          ./sh

      - name: Cache Flutter Dependencies
        uses: actions/cache@v3
        with:
          path: |
            ~/.pub-cache
            ${{ runner.temp }}/flutter
          key: ${{ runner.os }}-flutter-${{ hashFiles('**/pubspec.yaml') }}
          restore-keys: |
            ${{ runner.os }}-flutter-

      - name: Echo PATH for Debugging
        run: echo $env:PATH

      - name: Build Flutter Desktop App
        env:
          PATH: ${{ env.PATH }};${{ runner.temp }}\.cargo\bin
        run: |
          cd presentation
          flutter doctor -v
          flutter build windows --release --dart-define-from-file=release.json
```
### Actual results

After building the app for around 4 mins i am getting an error which i think it is related to the same reason why flutter doctor -v doesn't work, which is because of _where.exe_, but i tried to print the env path and the path for system32 where the _where.exe_ is located, is present. I couldn't find any solutions regarding CI/CD for Windows apps.

### Logs

```console
"Echo Process Environment Variables" step: 

Process PATH:
C:\Program Files\PowerShell\7;C:\Users\runneradmin
\AppData\Local\Pub\Cache/bin;C:\hostedtoolcache\wi
ndows\flutter\stable-3.19.6-x64/bin/cache/dart-sdk
/bin;C:\hostedtoolcache\windows\flutter\stable-3.1
9.6-x64/bin;C:\Program Files\MongoDB\Server\5.0\bi
n;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NS
IS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\h
ostedtoolcache\windows\stack\2.15.7\x64;C:\cabal\b
in;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\d
otnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;
C:\Program Files\R\R-4.4.0\bin\x64;C:\SeleniumWebD
rivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDrive
r\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program F
iles (x86)\sbt\bin;C:\Program Files (x86)\GitHub C
LI;C:\Program Files\Git\bin;C:\Program Files (x86)
\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows
\go\1.21.10\x64\bin;C:\hostedtoolcache\windows\Pyt
hon\3.9.[13](https://github.com/CodeStorming/spm_desktop/actions/runs/9357033600/job/25755835152#step:6:14)\x64\Scripts;C:\hostedtoolcache\windows\
Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\
3.0.7\x64\bin;C:\Program Files\OpenSSL\bin;C:\tool
s\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temu
rin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files
\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Micro
soft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\P
rogramData\docker-compose;C:\ProgramData\Chocolate
y\bin;C:\Windows\system32;C:\Windows;C:\Windows\Sy
stem32\Wbem;C:\Windows\System32\WindowsPowerShell\
v1.0\;C:\Windows\System32\OpenSSH\;C:\Program File
s\dotnet\;C:\Program Files\PowerShell\7\;C:\Progra
m Files\Microsoft\Web Platform Installer\;C:\Progr
am Files\Microsoft SQL Server\Client SDK\ODBC\170\
Tools\Binn\;C:\Program Files\Microsoft SQL Server\
150\Tools\Binn\;C:\Program Files (x86)\Windows Kit
s\10\Windows Performance Toolkit\;C:\Program Files
 (x86)\WiX Toolset v3.[14](https://github.com/CodeStorming/spm_desktop/actions/runs/9357033600/job/25755835152#step:6:15)\bin;C:\Program Files\Micr
osoft SQL Server\130\DTS\Binn\;C:\Program Files\Mi
crosoft SQL Server\140\DTS\Binn\;C:\Program Files\
Microsoft SQL Server\[15](https://github.com/CodeStorming/spm_desktop/actions/runs/9357033600/job/25755835152#step:6:16)0\DTS\Binn\;C:\Program File
s\Microsoft SQL Server\[16](https://github.com/CodeStorming/spm_desktop/actions/runs/9357033600/job/25755835152#step:6:17)0\DTS\Binn\;C:\Strawberry
\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\p
erl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools
\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Pr
ogram Files\CMake\bin;C:\ProgramData\chocolatey\li
b\maven\apache-maven-3.8.7\bin;C:\Program Files\Mi
crosoft Service Fabric\bin\Fabric\Fabric.Code;C:\P
rogram Files\Microsoft SDKs\Service Fabric\Tools\S
erviceFabricLocalClusterManager;C:\Program Files\n
odejs\;C:\Program Files\Git\cmd;C:\Program Files\G
it\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Pro
gram Files\GitHub CLI\;c:\tools\php;C:\Program Fil
es (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\
;C:\Program Files\Amazon\SessionManagerPlugin\bin\
;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program
 Files\Microsoft SQL Server\1[30](https://github.com/CodeStorming/spm_desktop/actions/runs/9357033600/job/25755835152#step:6:31)\Tools\Binn\;C:\Pro
gram Files\LLVM\bin;C:\Users\runneradmin\.dotnet\t
ools;C:\Users\runneradmin\.cargo\bin;C:\Users\runn
eradmin\AppData\Local\Microsoft\WindowsApps

"Build Flutter Desktop App" step :

[☠] Flutter (the doctor check crashed)
    X Due to an error, the doctor check did not complete. If the error message below is not helpful, please let us know about this issue at https://github.com/flutter/flutter/issues.
    X Error: Cannot find the executable for `where`. This can happen if the System32 folder (e.g. C:\Windows\System32 ) is removed from the PATH environment variable. Ensure that this is present and then try again after restarting the terminal and/or IDE.
    • #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
      #1      _WindowsUtils._which (package:flutter_tools/src/base/os.dart:488:7)
      #2      OperatingSystemUtils.which (package:flutter_tools/src/base/os.dart:92:31)
      #3      FlutterValidator._validateSdkBinary (package:flutter_tools/src/doctor.dart:666:52)
      #4      FlutterValidator._validateRequiredBinaries (package:flutter_tools/src/doctor.dart:653:47)
      #5      FlutterValidator.validate (package:flutter_tools/src/doctor.dart:567:7)
      #6      Doctor.startValidatorTasks.<anonymous closure> (package:flutter_tools/src/doctor.dart:272:72)
      #7      asyncGuard.<anonymous closure> (package:flutter_tools/src/base/async_guard.dart:[11](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:12)1:32)
      #8      _rootRun (dart:async/zone.dart:1399:13)
      #9      _CustomZone.run (dart:async/zone.dart:1301:19)
      #10     _runZoned (dart:async/zone.dart:1804:10)
      #11     runZonedGuarded (dart:async/zone.dart:1792:[12](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:13))
      #12     runZoned (dart:async/zone.dart:1744:12)
      #13     asyncGuard (package:flutter_tools/src/base/async_guard.dart:109:3)
      #14     Doctor.startValidatorTasks (package:flutter_tools/src/doctor.dart:264:9)
      #15     Doctor.diagnose (package:flutter_tools/src/doctor.dart:396:72)
      #16     DoctorCommand.runCommand (package:flutter_tools/src/commands/doctor.dart:49:48)
      #17     FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1765:12)
      <asynchronous suspension>
      #18     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:[13](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:14)89:27)
      <asynchronous suspension>
      #19     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
      <asynchronous suspension>
      #20     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
      <asynchronous suspension>
      #21     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:355:9)
      <asynchronous suspension>
      #22     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:[15](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:16)3:19)
      <asynchronous suspension>
      #23     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:295:5)
      <asynchronous suspension>
      #24     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:119:9)
      <asynchronous suspension>
      #25     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
      <asynchronous suspension>
      #26     main (package:flutter_tools/executable.dart:90:3)
      <asynchronous suspension>


[√] Windows Version (Installed version of Windows is version 10 or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Android\android-sdk
    • Platform android-34-ext8, build-tools 34.0.0
    • ANDROID_HOME = C:\Android\android-sdk
    • ANDROID_SDK_ROOT = C:\Android\android-sdk
    • Java binary at: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin\java
    X Could not determine java version
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Enterprise 2022 17.9.7)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Enterprise
    • Visual Studio Enterprise 2022 version 17.9.34902.65
    ! Unable to locate a Windows 10 SDK. If building fails, install the Windows 10 SDK in Visual Studio.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.20348.2461]
    • Chrome (web)      • chrome  • web-javascript • unknown

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 4 categories.

Building Windows application...                                 
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\CMakeFiles\d68f6[16](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:17)fe25895c47e368c836433d5fe\super_native_extensions.dll.rule;D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\CMakeFiles\0aa2c7a6fc773ceb7e0db91daf795391\super_native_extensions_plugin_cargokit.rule;D:\a\spm_desktop\spm_desktop\presentation\windows\flutter\ephemeral\.plugin_symlinks\super_native_extensions\windows\CMakeLists.txt' exited with code 1. [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]
Building Windows application...                                   245.7s
Build process failed.
```

### Flutter Doctor output

<details open>
<summary>Doctor output</summary>

```console

[☠] Flutter (the doctor check crashed)
    X Due to an error, the doctor check did not complete. If the error message below is not helpful, please let us know about this issue at https://github.com/flutter/flutter/issues.
    X Error: Cannot find the executable for `where`. This can happen if the System32 folder (e.g. C:\Windows\System32 ) is removed from the PATH environment variable. Ensure that this is present and then try again after restarting the terminal and/or IDE.
    • #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
      #1      _WindowsUtils._which (package:flutter_tools/src/base/os.dart:488:7)
      #2      OperatingSystemUtils.which (package:flutter_tools/src/base/os.dart:92:31)
      #3      FlutterValidator._validateSdkBinary (package:flutter_tools/src/doctor.dart:666:52)
      #4      FlutterValidator._validateRequiredBinaries (package:flutter_tools/src/doctor.dart:653:47)
      #5      FlutterValidator.validate (package:flutter_tools/src/doctor.dart:567:7)
      #6      Doctor.startValidatorTasks.<anonymous closure> (package:flutter_tools/src/doctor.dart:272:72)
      #7      asyncGuard.<anonymous closure> (package:flutter_tools/src/base/async_guard.dart:[11](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:12)1:32)
      #8      _rootRun (dart:async/zone.dart:1399:13)
      #9      _CustomZone.run (dart:async/zone.dart:1301:19)
      #10     _runZoned (dart:async/zone.dart:1804:10)
      #11     runZonedGuarded (dart:async/zone.dart:1792:[12](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:13))
      #12     runZoned (dart:async/zone.dart:1744:12)
      #13     asyncGuard (package:flutter_tools/src/base/async_guard.dart:109:3)
      #14     Doctor.startValidatorTasks (package:flutter_tools/src/doctor.dart:264:9)
      #15     Doctor.diagnose (package:flutter_tools/src/doctor.dart:396:72)
      #16     DoctorCommand.runCommand (package:flutter_tools/src/commands/doctor.dart:49:48)
      #17     FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1765:12)
      <asynchronous suspension>
      #18     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:[13](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:14)89:27)
      <asynchronous suspension>
      #19     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
      <asynchronous suspension>
      #20     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
      <asynchronous suspension>
      #21     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:355:9)
      <asynchronous suspension>
      #22     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:[15](https://github.com/CodeStorming/spm_desktop/actions/runs/9356586632/job/25754411567#step:11:16)3:19)
      <asynchronous suspension>
      #23     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:295:5)
      <asynchronous suspension>
      #24     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:119:9)
      <asynchronous suspension>
      #25     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
      <asynchronous suspension>
      #26     main (package:flutter_tools/executable.dart:90:3)
      <asynchronous suspension>


[√] Windows Version (Installed version of Windows is version 10 or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Android\android-sdk
    • Platform android-34-ext8, build-tools 34.0.0
    • ANDROID_HOME = C:\Android\android-sdk
    • ANDROID_SDK_ROOT = C:\Android\android-sdk
    • Java binary at: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin\java
    X Could not determine java version
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Enterprise 2022 17.9.7)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Enterprise
    • Visual Studio Enterprise 2022 version 17.9.34902.65
    ! Unable to locate a Windows 10 SDK. If building fails, install the Windows 10 SDK in Visual Studio.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.20348.2461]
    • Chrome (web)      • chrome  • web-javascript • unknown

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 4 categories.

```

</details>

**Comment by huycozy** [comment]
Hi @almightydreamer, do you execute Flutter command on `Windows\System32` path? Please check https://github.com/flutter/flutter/issues/119658#issuecomment-1412121745 to see if it helps. 

Please also check [Update your Windows PATH variable](https://docs.flutter.dev/get-started/install/windows/desktop?tab=download#update-your-windows-path-variable) as well.

**Comment by almightydreamer** [comment]
Hello, @huycozy, thanks for your reply.
I am not running flutter doctor on Windows/System32. Also all the path variables are good, they are shown in the logs

**Comment by huycozy** [comment]
Hi, 

>I am not running flutter doctor on Windows/System32

I meant your Flutter SDK location. I'm not sure which location `subosito/flutter-action@v2` is installed. Could you check it? 

Also, please try the latest Flutter stable version (3.22.1) to see if the issue persists or not.

**Comment by almightydreamer** [comment]
Hello,
at the moment I cannot migrate the project on 3.22.1 because of some issues on a device that this app needs to be running, but I tried it and same result.

After trying where.exe flutter and where.exe dart I got an error that neither of them could be found. I don't know why that's the reason because the flutter installed by subosito/flutter-action@v2 is present in the environment path variable

**Comment by amrgetment** [comment]
> Cannot find the executable for `where`

I think `which` is more cross-platform supported

**Comment by almightydreamer** [comment]
I decided to try a different approach for CI/CD and I used a simpler workflow and now the `flutter doctor -v` is looking good, but the problem building the app still persists
```yaml
jobs:
  build:
    name: Build
    runs-on: windows-latest

    outputs:
      app_version: ${{ steps.extract_version.outputs.app_version }}

    steps:
      - uses: actions/checkout@v3

      - name: Install Flutter SDK
        uses: subosito/flutter-action@v2
        with:
          flutter-version: '3.19.6'
          channel: "stable"

      - name: Install dependencies and generate models
        run: |
          ./sh

      - name: Echo PATH for Debugging
        run: echo $env:PATH

      - name: Build Flutter Desktop App
        shell: pwsh
        run: |
          cd presentation
          flutter doctor -v
          flutter build windows --release --dart-define-from-file=release.json
```



Build Flutter Desktop App step logs: 

```console
Run cd presentation
[√] Flutter (Channel stable, 3.1[9](https://github.com/CodeStorming/spm_desktop/actions/runs/9406925475/job/25911510456#step:7:10).6, on Microsoft Windows [Version 10.0.20348.2461], locale en-US)
    • Flutter version 3.19.6 on channel stable at C:\hostedtoolcache\windows\flutter\stable-3.19.6-x64
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 54e66469a9 (7 weeks ago), 2024-04-17 13:08:03 -0700
    • Engine revision c4cd48e186
    • Dart version 3.3.4
    • DevTools version 2.31.1

[√] Windows Version (Installed version of Windows is version [10](https://github.com/CodeStorming/spm_desktop/actions/runs/9406925475/job/25911510456#step:7:11) or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Android\android-sdk
    • Platform android-34-ext8, build-tools 34.0.0
    • ANDROID_HOME = C:\Android\android-sdk
    • ANDROID_SDK_ROOT = C:\Android\android-sdk
    • Java binary at: C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.4[12](https://github.com/CodeStorming/spm_desktop/actions/runs/9406925475/job/25911510456#step:7:13)-8\x64\bin\java
    X Could not determine java version
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Enterprise 2022 17.10.1)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Enterprise
    • Visual Studio Enterprise 2022 version 17.10.34928.[14](https://github.com/CodeStorming/spm_desktop/actions/runs/9406925475/job/25911510456#step:7:15)7
    • Windows 10 SDK version 10.0.22621.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.20348.2461]
    • Chrome (web)      • chrome  • web-javascript • unknown

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.

Building Windows application...                                 
  SEVERE: error[E0282]: type annotations needed for `Box<_>`
SEVERE : error : could not compile `time` (lib) due to 1 previous error [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]
SEVERE : warning : build failed, waiting for other jobs to finish... [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\CMakeFiles\d68f6[16](https://github.com/CodeStorming/spm_desktop/actions/runs/9406925475/job/25911510456#step:7:17)fe25895c47e368c836433d5fe\super_native_extensions.dll.rule;D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\CMakeFiles\0aa2c7a6fc773ceb7e0db91daf795391\super_native_extensions_plugin_cargokit.rule;D:\a\spm_desktop\spm_desktop\presentation\windows\flutter\ephemeral\.plugin_symlinks\super_native_extensions\windows\CMakeLists.txt' exited with code -1. [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]
Building Windows application...                                   396.0s
Build process failed.
```

**Comment by huycozy** [comment]
>  SEVERE: error[E0282]: type annotations needed for `Box<_>`
SEVERE : error : could not compile `time` (lib) due to 1 previous error [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]
SEVERE : warning : build failed, waiting for other jobs to finish... [D:\a\spm_desktop\spm_desktop\presentation\build\windows\x64\plugins\super_native_extensions\super_native_extensions_plugin_cargokit.vcxproj]

The error is related to `super_native_extensions` package and I see there is a similar error at https://github.com/superlistapp/super_native_extensions/issues/366. Looks like it's fixed on `super_native_extensions: ^0.8.15`. 

Closing this as it's not an issue on Flutter framework but on a 3rd party package instead.
